common/src/main/java/org/apache/uniffle/common/rpc/GrpcServer.java

```diff
@@ -40,24 +40,32 @@
 
 import org.apache.uniffle.common.config.RssBaseConf;
 import org.apache.uniffle.common.metrics.GRPCMetrics;
+import org.apache.uniffle.common.util.Constants;
 import org.apache.uniffle.common.util.ExitUtils;
+import org.apache.uniffle.common.util.RssUtils;
 import org.apache.uniffle.common.util.ThreadUtils;
 
 public class GrpcServer implements ServerInterface {
 
   private static final Logger LOG = LoggerFactory.getLogger(GrpcServer.class);
 
-  private final Server server;
+  private Server server;
   private final int port;
   private int listenPort;
   private final ExecutorService pool;
+  private List<Pair<BindableService, List<ServerInterceptor>>> servicesWithInterceptors;
+  private GRPCMetrics grpcMetrics;
+  private RssBaseConf rssConf;
 
   protected GrpcServer(
       RssBaseConf conf,
       List<Pair<BindableService, List<ServerInterceptor>>> servicesWithInterceptors,
       GRPCMetrics grpcMetrics) {
-    this.port = conf.getInteger(RssBaseConf.RPC_SERVER_PORT);
-    long maxInboundMessageSize = conf.getLong(RssBaseConf.RPC_MESSAGE_MAX_SIZE);
+    this.rssConf = conf;
+    this.port = rssConf.getInteger(RssBaseConf.RPC_SERVER_PORT);
```

Comment:
If using the random port, why reserve this var of `port` ? the `listen port` and `port` are unclear.

Reply:
agree with u.in same test case have reuse RssBaseConf(i think it is a mistake), retain 
 this the port var just reserve the configuration value to avoid test failed.
[ShuffleWithRssClientTest](https://github.com/apache/incubator-uniffle/blob/master/integration-test/common/src/test/java/org/apache/uniffle/test/ShuffleWithRssClientTest.java)